joern-cli/frontends/gosrc2cpg/src/main/scala/io/joern/gosrc2cpg/astcreation/AstForFunctionsCreator.scala

```diff
@@ -127,7 +126,7 @@ trait AstForFunctionsCreator(implicit withSchemaValidation: ValidationMode) { th
         val typeInfo                                           = createParserNodeInfo(x(ParserKeys.Type))
         val (typeFullName, typeFullNameForcode, isVariadic, _) = processTypeInfo(typeInfo, genericTypeMethodMap)
         x(ParserKeys.Names).arrOpt
-          .getOrElse(List())
+          .getOrElse(List(""))
```

Comment:
Why a blank string?

Reply:
While generating the signature for a function structure

```
func test(a, b int, c string) int {
}

```
it works because if there is no situation where there will not be a parameter name exist.

When I reuse the same function for generating the signature for lambda types as below

`type Operation func(int, int) int`

Now in this case there is no parameter name exist, in order to handle this situation add this empty string, which only facilitates adding the parameter type to list.

Reply:
Ok cool, maybe next PR it would be good to have a comment here, since someone debugging or developing in the future might be confused/accidentally delete it.